### ISSUE TITLE
Neo support

### ISSUE THREAD

**Issue by apdavison** [CINPLA/pyxona#13]
Hi all, I just came across some data in Axona format, which we would like to analyse with Elephant. Would you be interested in helping us wrap pyxona to provide a Neo IO module?

**Comment by lepmik** [comment]
Absolutely! pyxona is written with inspiration from Neo, as you might notice from the nomenclature. The main reason we did not write it directly in Neo is due to Neo's lack of support for tracking (position) data. However, I think it should be straight forward to have Neo wrap that as an analog signal? 

I would however not have the time in the near future (finishing PhD thesis, baby on the way and starting a new position) to contribute directly with coding etc, but I would be available for questions and bug fixes.
